### DISCUSSION TITLE
Feature Request: Don't block image play buttons while playing a video file

### DISCUSSION THREAD

**Discussion by avanzandvliet** [AntonyCorbett/OnlyM#459] [Ideas]
Hi,

We use OnlyM with much appreciation. I got a small "setttings" feature request.

We would like to be able to show an image, at the end of a video file. In the current situation, we first have to stop the playback of the video file, which results in the playback window going away. Showing JWLibrary underneath, or the desktop if JWLibrary is currectly not running. (Zoom in some cases)

We would like to be able, to have a setting, that the play buttons of images, are not greyed out, during playback of a video file, so that we can start the image when we need to, and that the program underneath is not shown.

I hope I explained the situation clearly, if this is not possible, or you do not have the time to make this, that is ok. Thank you for making OnlyM.

Greetings from Holland.

Alexander.

**Comment by AntonyCorbett** [comment]
@avanzandvliet Thanks for your request. How about using the permanent backdrop option in OnlyM?
